dockertest/images.py

```diff
@@ -54,11 +54,11 @@ class DockerImage(object):  # pylint: disable=R0902
     #: parsing, spec defined in docker-io documentation.  e.g.
     #: ``[registry_hostname[:port]/][user_name/]
     #: (repository_name[:version_tag])``
-    repo_split_p = re.compile(r"(?P<repo_addr>.+?"  # optional
-                              r"(?P<repo_addr_port>:[\d]+?)?/)?"  # optional
-                              r"(?P<user>[\w\-\.\+]+/)?"  # optional
-                              r"(?P<repo>[\w\-\.]+)"  # required
-                              r"(?P<tag>:[\w\-\.]+)?")  # optional
+    repo_split_p = (r"(?P<repo_addr>.+?"  # optional
+                    r"(?P<repo_addr_port>:[\d]+?)?/)?"  # optional
+                    r"(?P<user>[\w\-\.\+]+/)?"  # optional
+                    r"(?P<repo>[\w\-\.]+)"  # required
+                    r"(?P<tag>:[\w\-\.]*)?")  # optional
```

Comment:
The re.compile made it so I couldn't reuse this regex in events.py. If you know of a way to re.UNcompile, or a better way to do that code in events.py, I'm open to suggestion.